CHANGELOG.md

```diff
@@ -1,9 +1,31 @@
 # Changelog
 
+## Changelog Changes 
+
+The Hyperledger Besu Wiki includes a [proposal to change what's included in this changelog](https://wiki.hyperledger.org/display/BESU/Changes+to+Changelog).  
+The changelog from 1.3.6 is moving towards the proposed changelog approach.  If you've got feedback or 
+specific items you'd like to see included in the changelog, add your comments to the proposal. 
+
 ## Istanbul Compatibility
 
 For compatibility with Ethereum Istanbul upgrade, use v1.3.4 or later. 
 
+### 1.3.6 
+
+### Additions and Improvements 
+
+- Performance improvements: 
+  * Multithread Websockets to increase throughput [\#231](https://github.com/hyperledger/besu/pull/231) 
+  * NewBlockHeaders performance improvement [\#230](https://github.com/hyperledger/besu/pull/230) 
+- Documentation updates include: 
+  * [Configuring mining using the Stratum protocol](https://besu.hyperledger.org/en/latest/HowTo/Configure/Configure-Mining/)
+  * [ETC network command line options](https://besu.hyperledger.org/en/latest/Reference/CLI/CLI-Syntax/#network)
+
```

Comment:
Sorry, I think we missed the PR infor for changelog lines in these two very late PRs

```suggestion
- Hard Fork Support:
   * MuirGlacier for Ethereum Mainnet and Ropsten
   * Agharta for Kotti, and Mordor

```